numpy/testing/noseclasses.py

```diff
@@ -1,6 +0,0 @@
-"""
-Back compatibility noseclasses module. It will import the appropriate
-set of tools
-
-"""
-from .nose_tools.noseclasses import *
```

Comment:
Retain for backward compat?

Reply:
Yeah, this shouldn't have been removed. I'll put it back.